pyccel/ast/builtins.py

```diff
@@ -501,7 +477,7 @@ class PythonRange(Basic):
     Range(0, n, 1)
     """
 
-    def __new__(cls, *args):
+    def __init__(self, *args):
```

Comment:
This may be beyond the scope of this PR, but nonetheless... Passed in this way to the constructor, `args` is always a tuple. So why do we check its type below? I think we have a bug.

Reply:
You are correct, I will fix it. I don't think it's a bug, just unnecessary code